### PR TITLE
Updated more spoken forms

### PR DESCRIPTION
**Pull request by AndreasArvidsson**
## Checklist

- [ ] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)
- [ ] I have updated the [docs](https://github.com/cursorless-dev/cursorless/tree/main/docs) and [cheatsheet](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon/src/cheatsheet)
- [ ] I have not broken the cheatsheet

### CODE REVIEW

File: packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/inference/takeVestPastAfterAir.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 languageId: typescript
 command:
   version: 1
-  spokenForm: take vest past after air
+  spokenForm: take vest past air
```
Comment (pokey) [inline packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/inference/takeVestPastAfterAir.yml:4]: This one isn't correct; see payload

File: packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/bringStateFineToAfterBatt.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 languageId: typescript
 command:
-  spokenForm: bring state fine to after bat
+  spokenForm: bring state fine after state bat
```
Comment (pokey) [inline packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/bringStateFineToAfterBatt.yml:3]: This is correct, but I'm very curious where the original spoken form came from 😅

File: packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/chuckAfterHarp.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 languageId: plaintext
 command:
   version: 1
-  spokenForm: chuck after harp
+  spokenForm: chuck trailing harp
```
Comment (pokey) [inline packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/chuckAfterHarp.yml:4]: This seems wrong

File: packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/chuckAfterLineVest.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 languageId: plaintext
 command:
   version: 1
-  spokenForm: chuck after line vest
+  spokenForm: chuck trailing line vest
```
Comment (pokey) [inline packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/chuckAfterLineVest.yml:4]: And this one. Was there some migration we did where "chuck after" became "chuck trailing"?

File: packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/chuckAfterLook.yml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 languageId: plaintext
 command:
   version: 1
-  spokenForm: chuck after look
+  spokenForm: chuck trailing look
```
Comment (pokey) [inline packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/chuckAfterLook.yml:4]: As above. Looks like there's a few more

File: packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/surroundingPair/parseTree/python/chuckMatching.yml [hunk 6]
```diff
@@ -1,7 +1,7 @@
 languageId: python
 command:
   version: 1
-  spokenForm: change inside
+  spokenForm: change pair
```
Comment (pokey) [inline packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/surroundingPair/parseTree/python/chuckMatching.yml:4]: Weird 😅. Looks right tho

**Review by pokey — COMMENTED** [review]
I'm assuming this was your spoken form generator? Good stuff! Only a few misses

File: packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/inference/takeVestPastAfterAir.yml (hunk 1 above)
Comment (AndreasArvidsson) [reply packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/inference/takeVestPastAfterAir.yml:4]: In our migration we're just dropping before/after for actions where they don't make sense

File: packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/bringStateFineToAfterBatt.yml (hunk 2 above)
Comment (AndreasArvidsson) [reply packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/bringStateFineToAfterBatt.yml:3]: Yeah there are a few of those where this original spoken form is just weird

File: packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/chuckAfterHarp.yml (hunk 3 above)
Comment (AndreasArvidsson) [reply packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/chuckAfterHarp.yml:4]: This seems correct to me. We used to support chuck before and after and they did the same thing as chuck leading/trailing today.
https://github.com/cursorless-dev/cursorless/blob/6a78771b47308a8574a0fa46a50723c0f2c936e9/packages/cursorless-engine/src/core/commandVersionUpgrades/upgradeV1ToV2/upgradeV1ToV2.ts#L108-L114

File: packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/chuckAfterLineVest.yml (hunk 4 above)
Comment (AndreasArvidsson) [reply packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/chuckAfterLineVest.yml:4]: Yes
https://github.com/cursorless-dev/cursorless/blob/6a78771b47308a8574a0fa46a50723c0f2c936e9/packages/cursorless-engine/src/core/commandVersionUpgrades/upgradeV1ToV2/upgradeV1ToV2.ts#L108-L114

File: packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/surroundingPair/parseTree/python/chuckMatching.yml (hunk 6 above)
Comment (AndreasArvidsson) [reply packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/surroundingPair/parseTree/python/chuckMatching.yml:4]: Definitely. If you look at the file name. This isn't even the original spoken form.

**Comment by AndreasArvidsson** [comment]
> I'm assuming this was your spoken form generator? 

Correct.

File: packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/inference/takeVestPastAfterAir.yml (hunk 1 above)
Comment (pokey) [reply packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/inference/takeVestPastAfterAir.yml:4]: Interesting. We should prob change this test to use "end of". I'll file; I think it's out of scope here

**Review by pokey — APPROVED** [review]

File: packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/inference/takeVestPastAfterAir.yml (hunk 1 above)
Comment (AndreasArvidsson) [reply packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/inference/takeVestPastAfterAir.yml:4]: Should we just do this in our migrations instead? tale after > take end of.

**Pull request merged**